packages/manager/apps/telecom/src/app/telecom/telephony/service/contact/contact.service.js

```diff
@@ -0,0 +1,96 @@
+import map from 'lodash/map';
+import set from 'lodash/set';
+
+export default class ContactService {
+  /* @ngInject */
+  constructor($http, $q) {
+    this.$http = $http;
+    this.$q = $q;
+  }
+
+  getDirectory(billingAccount, serviceName) {
+    return this.$http
+      .get(`/telephony/${billingAccount}/service/${serviceName}/directory`)
+      .then(({ data }) => data)
+      .catch(({ data }) => this.$q.reject(data.message));
+  }
+
+  putDirectory(billingAccount, serviceName, params) {
+    return this.$http
+      .put(
+        `/telephony/${billingAccount}/service/${serviceName}/directory`,
+        params,
+      )
+      .then(({ data }) => data)
+      .catch(({ data }) => this.$q.reject(data.message));
+  }
+
+  fetchEntrepriseInformations(billingAccount, serviceName, entrepriseNumber) {
+    return this.$http
+      .post(
+        `/telephony/${billingAccount}/service/${serviceName}/directory/fetchEntrepriseInformations`,
+        {
+          entrepriseNumber,
+        },
+      )
+      .then(({ data }) => {
+        return data;
+      })
```

Comment:
To be consistent with others service function:
```suggestion
      .then(({ data }) => data)
```

Reply:
it's applied